### ISSUE TITLE
Deploying the project locally 

### ISSUE THREAD

**Issue by codr17** [jReddit/jReddit#130]
Hi,

I want to contribute to the project and implement some methods. I was able to build the project using `gradle build`. However I'm not sure what to do after that? Can I deploy the project locally if I want to test it using a REST client like Postman? When I imported it as a gradle project in Eclipse, it doesn't give me an option to run it locally via Tomcat. 

Thanks

**Comment by sfat** [comment]
Hi @codr17 ,
The thing is that this library doesn't work as a web server. It's just a java library that let's you interact with the Reddit API through some methods.  The requests are made through Java behind the scene inside the jReddit library. All you need to do is take what it tells you on the front page of this repo, login and create your restclient and whatnot and call some java methods (i.e. ```getSubmissions()```) and you'll get the response from Reddit. 
The thing that you need to do is to have a REST client like Postman, or whatever and see what information you get from Reddit API and map it to Java concrete models or methods and return information through Java methods.

**Comment by codr17** [comment]
Ok so let me reiterate what you're saying to make sure I understand: to test any methods I implement, or the methods that have already been implemented, you suggest I create a test class of my own and call the methods from this API to see if they're implemented properly?

Also, what's the use of Sonatype nexus? I saw the plugin in the Pom.xml file, so I thought maybe that's being used for deployment of the library.
